### PR TITLE
Feature/scala style for Spark

### PR DESCRIPTION
**Pull request by GezimSejdiu**
This PR makes the whole module for Spark as Scala-style compliant.

Best,

### CODE REVIEW

**Comment by GezimSejdiu** [comment]
Do not merge this one until I finish for the common layer as well. Will do it soon :)

**Comment by GezimSejdiu** [comment]
Many thanks, @patrickwestphal for pointing it out. I will change these methods. The good thing is that the implicits will not be affected. 

I also did a Scala-style compliant for the common layer of SANSA. So if you feel it is ready to be merged you can do it.

Many thanks in advance,

File: sansa-rdf-common/src/main/scala/net/sansa_stack/rdf/common/partition/utils/RdfTerm.scala [hunk 1]
```diff
@@ -11,31 +10,30 @@ import org.apache.jena.graph.Node
 case class RdfTerm(t: Int, v: String, lang: String, dt: String)
 
 object RdfTerm {
-  def toLexicalForm(o: Any) = "" + o //NodeFmtLib.str(node)
+  def toLexicalForm(o: Any): String = "" + o // NodeFmtLib.str(node)
 
-  def termToNode(term: RdfTerm) = {
+  def termToNode(term: RdfTerm): Unit = {
```
Comment (patrickwestphal) [inline sansa-rdf-common/src/main/scala/net/sansa_stack/rdf/common/partition/utils/RdfTerm.scala]: Is the return type correct? Shouldn't it be `Node` ?

Comment (GezimSejdiu) [reply sansa-rdf-common/src/main/scala/net/sansa_stack/rdf/common/partition/utils/RdfTerm.scala]: Indeed, it should be `Node` instead. Autocomplete did the job:). Will fix it now! 
Thanks for it, Patrick.

Comment (GezimSejdiu) [reply sansa-rdf-common/src/main/scala/net/sansa_stack/rdf/common/partition/utils/RdfTerm.scala]: Done!

**Pull request merged**